packages/react-internal/src/components/Panel/Panel.base.tsx

```diff
@@ -411,8 +411,9 @@ export class PanelBase extends React.Component<IPanelProps, IPanelState> impleme
     if (this.isActive && panel && !ev.defaultPrevented) {
       if (!elementContains(panel, ev.target as HTMLElement)) {
         if (this.props.onOuterClick) {
-          this.props.onOuterClick();
-          ev.preventDefault();
+          if (!this.props.onOuterClick()) {
+            ev.preventDefault();
```

Comment:
@tacambridge - do you know why we added `ev.preventDefault()` here in #5422?

I think the right thing to do is to pass `ev` to `onOuterClick` and let user decide what to do with the event, simply:
```
if (this.props.onOuterClick) {
  this.props.onOuterClick(ev)
}
```

Reply:
Makes sense, I updated and tested this, and the behavior is the same!